scene/gui/control.cpp

```diff
@@ -2724,6 +2724,29 @@ void Control::_window_find_focus_neighbor(const Vector2 &p_dir, Node *p_at, cons
 	}
 }
 
+void Control::set_autofocus(bool p_enable) {
+	if (data.autofocus == p_enable) {
+		return;
+	}
+	data.autofocus = p_enable;
+}
+
+bool Control::get_autofocus() const {
+	return data.autofocus;
+}
+
+void Control::attempt_autofocus() {
+#ifdef TOOLS_ENABLED
+	if (Engine::get_singleton()->is_editor_hint()) {
```

Comment:
There's no reason to disallow using it in the editor UI.

```suggestion
	if (is_part_of_edited_scene()) {
```

Reply:
@gmikhail Bump :slightly_smiling_face: Can you look into applying this suggestion?